src/providers/index.ts

```diff
@@ -47,6 +47,7 @@ import { UpstageConfig } from './upstage';
 import { LAMBDA } from '../globals';
 import { LambdaProviderConfig } from './lambda';
 import { DashScopeConfig } from './dashscope';
+import xAIConfig from './xAI';
```

Comment:
please follow PascalCase where required (you can refer look up other providers for reference)